### ISSUE TITLE
Unable to switch PHP version from 7.4

### ISSUE THREAD

**Issue by brianhogg** [stellarwp/slic#187]
## Steps to reproduce

1. Run `slic php-version set 8.1`
2. Response `yes` to restart the stack now

## Expected

Stack images are restarted successfully

## Actual

```
PHP version set to 8.2
Do you want to restart the stack now?  (yes) yes
Building the stack images...



Stack images built.

Updating the stack images...


[+] Pulling 5/0
 ✘ db Error        0.0s
 ✘ slic Error      0.0s
 ✘ wordpress Error 0.0s
 ✘ redis Error     0.0s
 ✘ chrome Error    0.0s
Error response from daemon: Get "https://ghcr.io/v2/": dialing ghcr.io:443 with direct connection: connecting to 140.82.112.34:443: dial tcp 140.82.112.34:443: connect: bad file descriptor
```

https://ghcr.io/v2/ appears to require auth:

```
{"errors":[{"code":"UNAUTHORIZED","message":"authentication required"}]}
```

**Comment by borkweb** [comment]
Hrm, that's curious! I haven't seen that with slic before, so this will be a good thing to nail down and add to our docs! 🤔 

Are you able to fetch the docker image within whatever environment you are running slic in?

```bash
wget https://ghcr.io/stellarwp/slic-wordpress-php8.2
```

**Comment by brianhogg** [comment]
While setting it up with a different plugin it seems to have used the 8.2 image. So I guess it needs to be done prior to doing anything else, or else deleting the docker images like @ntomkin mentioned.

After install I just get the following after doing a successful `slic init <plugin>`:

```
slic run
slic version 1.6.1 - StellarWP local testing and development tool

Using lifterlms


In Configuration.php line 236:

  Output path is not defined by key "paths: output"
```

I can open up a new issue for the above if you'd like. Thanks!

**Comment by cliffordp** [comment]
Just tried it with slic v1.6.1 and `slic php-version set 8.1` worked for me (was previously `8.0` before this command)
